.github/workflows/docker.yml

```diff
@@ -0,0 +1,96 @@
+name: Docker
+
```

Comment:
First, this file is inspirational, thank you!

Let me think about how to dockerize the whole app, hold on.